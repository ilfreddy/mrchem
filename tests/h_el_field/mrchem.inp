# vim:syntax=sh:

abs_prec = 1.0e-4                 # Overall relative precision

MRA {
  min_scale = -4                  # Root scale defines world size 2^{-n}
  corner = [ -1, -1, -1 ]         # Translation of corner root box [x,y,z]
  boxes = [ 2, 2, 2 ]             # Number of root boxes [x,y,z]
}

Molecule {
  multiplicity = 2
$coords
H       0.0000  0.0000  0.0000
$end
}

WaveFunction {
  method = HF                     # Wave function method (HF or DFT)
  restricted = false
}

ExternalField {
  electric_run = true
  electric_field = [ 0.0, 0.0, 0.01 ]
}

Properties {
  scf_energy = true               # Compute ground state energy
  dipole_moment = true            # Compute electrostatic dipole moment
}

SCF {
  kain = 3                        # Length of KAIN iterative history
  max_iter = 5
  initial_guess = CORE_DZ
}

