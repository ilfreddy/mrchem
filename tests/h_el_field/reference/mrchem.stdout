


************************************************************
***     __  __ ____   ____ _                             ***
***    |  \/  |  _ \ / ___| |__   ___ _ __ ___           ***
***    | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \          ***
***    | |  | |  _ <| |___| | | |  __/ | | | | |         ***
***    |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|         ***
***                                                      ***
***    VERSION 0.0.1 (rev. ab0790d0)                      ***
***                                                      ***
***    Stig Rune Jensen <stig.r.jensen@uit.no>           ***
***    Luca Frediani    <luca.frediani@uit.no>           ***
***    Peter Wind       <peter.wind@uit.no>              ***
***                                                      ***
************************************************************


+++ Serial execution


------------------------------------------------------------

 MRCPP version   : X.Y.Z
 Git revision    : 735162b

 Linear algebra  : EIGEN
 Parallelization : NONE

------------------------------------------------------------




============================================================
                  MultiResolution Analysis                  
------------------------------------------------------------
 polynomial order = 5
 polynomial type  = Interpolating
------------------------------------------------------------
 unit length      = 16.000000
 total boxes      = 8
 boxes            = [           2           2           2 ]
 lower bounds     = [  -16.000000  -16.000000  -16.000000 ]
 upper bounds     = [   16.000000   16.000000   16.000000 ]
 total length     = [   32.000000   32.000000   32.000000 ]
============================================================


============================================================
                          Molecule
------------------------------------------------------------
 Nr  Element             x             y             z      
------------------------------------------------------------
  1     H           0.00000e+00   0.00000e+00   0.00000e+00
------------------------------------------------------------
 Center of mass:    0.00000e+00   0.00000e+00   0.00000e+00
============================================================


============================================================
                Setting up nuclear potential
------------------------------------------------------------
 Nr  Element         Charge        Precision     Smoothing 
------------------------------------------------------------
  1     H            1.00000e+00   1.00000e-03   1.63241e-02
============================================================


============================================================
                Setting up nuclear potential
------------------------------------------------------------
 Nr  Element         Charge        Precision     Smoothing 
------------------------------------------------------------
  1     H            1.00000e+00   1.00000e-03   1.63241e-02
============================================================


============================================================
                  Projecting Hydrogen AOs
------------------------------------------------------------
    N    Atom   Label                     SquareNorm
------------------------------------------------------------
    1     H1     1s                   1.000001700982792e+00
    2     H1     2s                   9.998015962401137e-01
    3     H1     2p                   9.999164742286865e-01
    4     H1     2p                   9.999164742286865e-01
    5     H1     2p                   9.999164742286863e-01
------------------------------------------------------------
                 Wall time: 1.01763e-01 sec 
============================================================


============================================================
            Diagonalize Core-Hamiltonian matrix
------------------------------------------------------------
 Nuclear potential                     712      8.93002e-02
 Compute Fock matrix                            4.55736e-01
 Diagonalize Fock matrix                        1.06150e-05
 Rotate orbitals                                2.01116e-03
------------------------------------------------------------
                 Wall time: 4.57848e-01 sec 
============================================================


============================================================
                        Eigenvalues
------------------------------------------------------------
-4.9975869010e-01
-1.2485455524e-01
-1.2485455524e-01
-1.2485455524e-01
-1.2463039627e-01
============================================================


============================================================
 OrbitalVector:   1 orbitals     1 occupied     1 electrons
------------------------------------------------------------
   n  RankID           Norm          Spin Occ      Error    
------------------------------------------------------------
   0    -1       1.000000000000e+00    a   1    1.00000e+00
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Nuclear potential                     712      8.84639e-02
 Coulomb density                       264      3.31538e-02
 Coulomb potential                       8      1.09679e-01
 Hartree-Fock exchange                 264      6.48239e-01
------------------------------------------------------------
                 Wall time: 8.81462e-01 sec 
============================================================


============================================================
                  Calculating Fock matrix
------------------------------------------------------------
 Kinetic part                         5.562855600000000e-02
 Potential part                       1.513676300000000e-01
------------------------------------------------------------
                 Wall time: 2.07064e-01 sec 
============================================================




####################### SCF cycle  1 #######################


============================================================
 Current precision                              1.00000e-03
------------------------------------------------------------
 Orbital threshold                              1.00000e-02
 Property threshold                             1.00000e-04
============================================================


============================================================
                  Digonalizing Fock matrix
------------------------------------------------------------
 Computing Lowdin matrix                        2.05883e-04
 Diagonalizing matrix                           2.09600e-06
------------------------------------------------------------
                 Wall time: 5.82999e-03 sec 
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                       1.446993660248090e-16
 Electronic energy                   -5.005300224259814e-01
------------------------------------------------------------
 Total energy                        -5.005300224259813e-01
------------------------------------------------------------
                 Wall time: 1.47900e-01 sec 
============================================================


============================================================
               Setting up Helmholtz operators
------------------------------------------------------------
 Creating operator with lambda                 -5.01301e-01
------------------------------------------------------------
                 Wall time: 6.02038e-02 sec 
============================================================


============================================================
               Setting up Helmholtz arguments
------------------------------------------------------------
 Potential part                                 1.49640e-01
 Matrix part                                    7.27000e-06
------------------------------------------------------------
   0       2.23543e-01    264            0.00000e+00      0
------------------------------------------------------------
                 Wall time: 1.51816e-01 sec 
============================================================


============================================================
                Applying Helmholtz operators
------------------------------------------------------------
 Orb    RealNorm   Nodes     ImagNorm   Nodes     Timing
------------------------------------------------------------
  0    1.00008e+00   200    0.00000e+00     0   1.09140e+00
------------------------------------------------------------
                 Wall time: 1.09144e+00 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             1.47800e-06
 Copy orbitals                                  1.32484e-03
 Copy orbital updates                           1.27482e-03
------------------------------------------------------------
                 Wall time: 2.65375e-03 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Nuclear potential                     712      8.47417e-02
 Coulomb density                       264      2.86156e-02
 Coulomb potential                       8      9.71854e-02
 Hartree-Fock exchange                 264      6.57415e-01
------------------------------------------------------------
                 Wall time: 8.69545e-01 sec 
============================================================


============================================================
                  Calculating Fock matrix
------------------------------------------------------------
 Kinetic part                         4.453721200000000e-02
 Potential part                       1.400054950000000e-01
------------------------------------------------------------
                 Wall time: 1.84616e-01 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -5.01600e-01   1.24597e-02        200    a    1    0
------------------------------------------------------------
 Total error:                            1.24597e-02    0
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Orbital      -5.013013547492868e-01    -5.01301e-01    0
 Kinetic       5.003116191124547e-01     5.00312e-01    0
 N-E          -1.000070309215131e+00    -1.00007e+00    0
 Coulomb       3.116875163759140e-01     3.11688e-01    0
 Exchange     -3.124588486992195e-01    -3.12459e-01    0
 X-C           0.000000000000000e+00     0.00000e+00    1
------------------------------------------------------------
 Electronic   -5.005300224259814e-01    -5.00530e-01    0
 Nuclear       1.446993660248090e-16     1.44699e-16    1
------------------------------------------------------------
 Total        -5.005300224259813e-01    -5.00530e-01    0
============================================================


################ Wall time: 2.53076e+00 sec ################




####################### SCF cycle  2 #######################


============================================================
 Current precision                              1.00000e-03
------------------------------------------------------------
 Orbital threshold                              1.00000e-02
 Property threshold                             1.00000e-04
============================================================


============================================================
                  Digonalizing Fock matrix
------------------------------------------------------------
 Computing Lowdin matrix                        2.08222e-04
 Diagonalizing matrix                           2.34700e-06
------------------------------------------------------------
                 Wall time: 3.59299e-03 sec 
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                       1.446993660248090e-16
 Electronic energy                   -5.007850727332238e-01
------------------------------------------------------------
 Total energy                        -5.007850727332237e-01
------------------------------------------------------------
                 Wall time: 1.45321e-01 sec 
============================================================


============================================================
               Setting up Helmholtz operators
------------------------------------------------------------
 Creating operator with lambda                 -5.01600e-01
------------------------------------------------------------
                 Wall time: 5.65492e-02 sec 
============================================================


============================================================
               Setting up Helmholtz arguments
------------------------------------------------------------
 Potential part                                 1.44576e-01
 Matrix part                                    6.84600e-06
------------------------------------------------------------
   0       2.23200e-01    264            0.00000e+00      0
------------------------------------------------------------
                 Wall time: 1.46499e-01 sec 
============================================================


============================================================
                Applying Helmholtz operators
------------------------------------------------------------
 Orb    RealNorm   Nodes     ImagNorm   Nodes     Timing
------------------------------------------------------------
  0    1.00010e+00   200    0.00000e+00     0   1.07840e+00
------------------------------------------------------------
                 Wall time: 1.07844e+00 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             1.24700e-06
 Copy orbitals                                  1.26198e-03
 Copy orbital updates                           1.27190e-03
------------------------------------------------------------
                 Wall time: 2.57863e-03 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Nuclear potential                     712      8.40099e-02
 Coulomb density                       264      2.81289e-02
 Coulomb potential                       8      9.92620e-02
 Hartree-Fock exchange                 264      6.62026e-01
------------------------------------------------------------
                 Wall time: 8.75059e-01 sec 
============================================================


============================================================
                  Calculating Fock matrix
------------------------------------------------------------
 Kinetic part                         4.568021000000001e-02
 Potential part                       1.426739850000000e-01
------------------------------------------------------------
                 Wall time: 1.88425e-01 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -5.01654e-01   5.66677e-03        200    a    1    1
------------------------------------------------------------
 Total error:                            5.66677e-03    1
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Orbital      -5.016003227852797e-01    -2.98968e-04    0
 Kinetic       5.006629899034947e-01     3.51371e-04    0
 N-E          -1.000400384634753e+00    -3.30075e-04    0
 Coulomb       3.121804735144466e-01     4.92957e-04    0
 Exchange     -3.129957235665024e-01    -5.36875e-04    0
 X-C           0.000000000000000e+00     0.00000e+00    1
------------------------------------------------------------
 Electronic   -5.007850727332238e-01    -2.55050e-04    0
 Nuclear       1.446993660248090e-16     0.00000e+00    1
------------------------------------------------------------
 Total        -5.007850727332237e-01    -2.55050e-04    0
============================================================


################ Wall time: 2.51294e+00 sec ################




####################### SCF cycle  3 #######################


============================================================
 Current precision                              1.00000e-03
------------------------------------------------------------
 Orbital threshold                              1.00000e-02
 Property threshold                             1.00000e-04
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                       1.446993660248090e-16
 Electronic energy                   -5.008304439708937e-01
------------------------------------------------------------
 Total energy                        -5.008304439708936e-01
------------------------------------------------------------
                 Wall time: 1.45132e-01 sec 
============================================================


============================================================
               Setting up Helmholtz operators
------------------------------------------------------------
 Creating operator with lambda                 -5.01654e-01
------------------------------------------------------------
                 Wall time: 5.64997e-02 sec 
============================================================


============================================================
               Setting up Helmholtz arguments
------------------------------------------------------------
 Potential part                                 1.49617e-01
 Matrix part                                    7.78200e-06
------------------------------------------------------------
   0       2.23126e-01    264            0.00000e+00      0
------------------------------------------------------------
                 Wall time: 1.51878e-01 sec 
============================================================


============================================================
                Applying Helmholtz operators
------------------------------------------------------------
 Orb    RealNorm   Nodes     ImagNorm   Nodes     Timing
------------------------------------------------------------
  0    1.00016e+00   200    0.00000e+00     0   1.11130e+00
------------------------------------------------------------
                 Wall time: 1.11134e+00 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             1.25000e-06
 Setup linear system                            3.76320e-03
 Solve linear system                            4.03400e-06
 Expand solution                                4.93724e-03
------------------------------------------------------------
                 Wall time: 8.78282e-03 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Nuclear potential                     712      8.67823e-02
 Coulomb density                       264      2.85785e-02
 Coulomb potential                       8      9.93271e-02
 Hartree-Fock exchange                 264      6.58865e-01
------------------------------------------------------------
                 Wall time: 8.75168e-01 sec 
============================================================


============================================================
                  Calculating Fock matrix
------------------------------------------------------------
 Kinetic part                         4.535223800000000e-02
 Potential part                       1.448068040000000e-01
------------------------------------------------------------
                 Wall time: 1.90230e-01 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -5.01672e-01   5.09427e-03        200    a    1    1
------------------------------------------------------------
 Total error:                            5.09427e-03    1
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Orbital      -5.016544646414942e-01    -5.41419e-05    0
 Kinetic       5.008025635542631e-01     1.39574e-04    0
 N-E          -1.000467030410574e+00    -6.66458e-05    1
 Coulomb       3.123073871506888e-01     1.26914e-04    0
 Exchange     -3.131314078212893e-01    -1.35684e-04    0
 X-C           0.000000000000000e+00     0.00000e+00    1
------------------------------------------------------------
 Electronic   -5.008304439708937e-01    -4.53712e-05    1
 Nuclear       1.446993660248090e-16     0.00000e+00    1
------------------------------------------------------------
 Total        -5.008304439708936e-01    -4.53712e-05    1
============================================================


################ Wall time: 2.55057e+00 sec ################


============================================================
                      Convergence rate
------------------------------------------------------------
Iter    OrbError       Property                   Update  
------------------------------------------------------------
  1    1.24597e-02    -5.005300224259813e-01   -5.00530e-01
  2    5.66677e-03    -5.007850727332237e-01   -2.55050e-04
  3    5.09427e-03    -5.008304439708936e-01   -4.53712e-05
------------------------------------------------------------
                      SCF converged!!!                      
============================================================


============================================================
                  Digonalizing Fock matrix
------------------------------------------------------------
 Computing Lowdin matrix                        2.03442e-04
 Diagonalizing matrix                           2.08300e-06
------------------------------------------------------------
                 Wall time: 5.14315e-03 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Nuclear potential                     712      8.56710e-02
 Coulomb density                       264      2.75754e-02
 Coulomb potential                       8      1.00584e-01
 Hartree-Fock exchange                 264      6.71233e-01
------------------------------------------------------------
                 Wall time: 8.86658e-01 sec 
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
------------------------------------------------------------
                 Wall time: 1.76734e-01 sec 
============================================================


============================================================
                 Calculating dipole moment
------------------------------------------------------------
------------------------------------------------------------
                 Wall time: 9.99611e-02 sec 
============================================================


============================================================
                        Fock matrix
------------------------------------------------------------
-5.01672e-01
============================================================


============================================================
                      Orbital energies
------------------------------------------------------------
    n  spin  occ                            epsilon  
------------------------------------------------------------
    0    a    1                      -5.016720627637909e-01
============================================================


============================================================
                          Molecule
------------------------------------------------------------
 Nr  Element             x             y             z      
------------------------------------------------------------
  1     H           0.00000e+00   0.00000e+00   0.00000e+00
------------------------------------------------------------
 Center of mass:    0.00000e+00   0.00000e+00   0.00000e+00
============================================================


                                                            
============================================================
                         SCF Energy                         
------------------------------------------------------------
                                                            
 Sum orbital energy:                 -5.016720627637909e-01
 Kinetic energy:                      5.008101223315435e-01
 E-N energy:                         -1.000385114473555e+00
 Coulomb energy:                      3.123333829701325e-01
 Exchange energy:                    -3.131608455702758e-01
 X-C energy:                          0.000000000000000e+00
 El. External field energy:          -4.421454214925479e-04
 Nuc. External field energy:          1.446993660248090e-16
                                                            
------------------------------------------------------------
                                                            
 Electronic energy:                  -5.008446001636475e-01
 Nuclear energy:                      1.446993660248090e-16
                                                            
------------------------------------------------------------
                                                            
 Total energy       (au)             -5.008446001636474e-01
                    (kJ/mol)         -1.314967307408708e+03
                    (kcal/mol)       -3.142847291002077e+02
                    (eV)             -1.362867526526630e+01
                                                            
============================================================
                                                            

                                                            
============================================================
                         Dipole moment                      
------------------------------------------------------------
                                                            
 Length of vector:   (au)                  4.4214542149e-02
                     (Debye)               1.1238214587e-01
                                                            
------------------------------------------------------------
                                                            
  -2.7944423169e-15   -9.9737214730e-15    4.4214542149e-02
                                                            
============================================================
                                                            



************************************************************
***                                                      ***
***                    Exiting MRChem                    ***
***                                                      ***
***               Wall time: 1.048332e+01                ***
***                                                      ***
************************************************************



Precision is set by energy estimate Z^(5/2)
   est_norm   1.0
   abs_prec   0.0001
   rel_prec   0.001
